### PR TITLE
[FIX] Task hash's, added transforms to the hashing

### LINKED ISSUE #228: [FIX] Task hash's are now wrong
Based on the action+query+transform combo, task hash's are now breaking/obsolete.

New Formula:
```
let message = format!(
            "{:?}{:?}{:?}{:?}{:?}{:?}",
            self.owner_id, self.interval, self.boundary, self.actions, self.queries, self.transforms
        );
```

### PR DESCRIPTION
**Pull request by Buckram123**
Closes #228.

### CODE REVIEW

**Review by TrevorJTClarke — APPROVED** [review]
THANK YOU SO MUCH!!!!!!!!!!!!!!! 😻

File: packages/cw-croncat-core/src/types.rs [hunk 1]
```diff
@@ -360,8 +360,13 @@ impl Task {
...
         let message = format!(
-            "{:?}{:?}{:?}{:?}{:?}",
-            self.owner_id, self.interval, self.boundary, self.actions, self.queries
+            "{:?}{:?}{:?}{:?}{:?}{:?}",
+            self.owner_id,
+            self.interval,
+            self.boundary,
+            self.actions,
+            self.queries,
+            self.transforms
```
Comment (mikedotexe) [inline packages/cw-croncat-core/src/types.rs:369]: 🚀

**Review by mikedotexe — APPROVED** [review]

**Comment by mikedotexe** [comment]
@Buckram123 looks like we have some minor merge conflicts, but it's ready to ship!

**Comment by Buckram123** [comment]
> @Buckram123 looks like we have some minor merge conflicts, but it's ready to ship!

fixing!

**Pull request merged**